src/include/network/widgets.rb

```diff
@@ -559,46 +559,48 @@ def handleDevice(items, selected)
     # @param [Array<Fixnum>] itemIds           list of indexes into LanItems::Items
     # @param [Array<String>] enslavedIfaces    list of device names of already enslaved devices
     def CreateSlaveItems(itemIds, enslavedIfaces)
-      itemIds = deep_copy(itemIds)
-      enslavedIfaces = deep_copy(enslavedIfaces)
+      raise ArgumentError if itemIds.nil?
```

Comment:
please specify message for argument error, this way it is quite useless, especially if you use more arguments constrains.